lib/static/modules/reducer.js

```diff
@@ -132,6 +132,9 @@ function reducer(state = getInitialState(compiledData), action) {
                 }
             };
         }
+        case actionNames.STOP_SERVER: {
+            return {...state, serverStopped: true};
```

Comment:
you should set to `false` options as `running` and `processing`, because you can stop server while tests executing